client/web/src/enterprise/batches/detail/changesets/HiddenExternalChangesetNode.tsx

```diff
@@ -5,10 +5,26 @@ import { HiddenExternalChangesetInfoCell } from './HiddenExternalChangesetInfoCe
 
 export interface HiddenExternalChangesetNodeProps {
     node: Pick<HiddenExternalChangesetFields, 'id' | 'nextSyncAt' | 'updatedAt' | 'state' | '__typename'>
+    enableSelect?: boolean
 }
 
-export const HiddenExternalChangesetNode: React.FunctionComponent<HiddenExternalChangesetNodeProps> = ({ node }) => (
+export const HiddenExternalChangesetNode: React.FunctionComponent<HiddenExternalChangesetNodeProps> = ({
+    node,
+    enableSelect = false,
+}) => (
     <>
+        {enableSelect && (
+            <div className="p-2">
+                <input
+                    id={`select-changeset-${node.id}`}
+                    type="checkbox"
+                    className="btn"
+                    checked={false}
+                    disabled={true}
+                    data-tooltip="You can't select this changeset to detach it due to lacking permissions"
```

Comment:
I think we can tighten that a touch:

```suggestion
                    data-tooltip="You do not have permission to detach this changeset"
```

Reply:
Thank you!